ieeeuoft/hackathon-template
# Ieee 193 simple order fulfillment

## Overview

- Resolves IEEE-195 and IEEE-193
- The Table in IEEE-193 is not ready to be implemented with the current state of our API
- for NewHacks we'll stick with simple order fulfillment
- this PR also adds the return order table (pretty much copied from participant dashboard)


## Unit Tests Created

- Checkbox tests for the table


## Steps to QA

- Go to /teams/<team code> and play around with the order tables
- check that the tables on the participant dashboard are unaffected
